cmd/util/ledger/migrations/storage_v6.go

```diff
@@ -1027,6 +1027,12 @@ func (c *ValueConverter) Convert(value oldInter.Value) (result newInter.Value) {
 		panic("converted value is nil")
 	}
 
+	switch expectedType {
+	case newInter.PrimitiveStaticTypeUInt64:
+		if intValue, ok := c.result.(newInter.IntValue); ok {
+			c.result = newInter.ConvertUInt64(intValue)
+		}
+	}
```

Comment:
Why is this handled specifically? Is the data somehow wrong? (Static type is `UInt64`, but value is `Int`?)

Reply:
There are few values, where the expected type is `UInt64` but the value is `Int`. These potentially are the ones where the type is changed when the contract is updated.
There are more similar ones in testnet, but not fixable. e.g.: expected `String` but found `UInt64`, etc.

Reply:
Ah, I see, this could come from before we had the contract update validation, and before we had the container mutation, makes sense 👍